### PR TITLE
Added Delete function to AzureSets

### PR DESCRIPTION
**Pull request by DYDR**

### CODE REVIEW

File: addons/azureSets/azuresets.lua [hunk 1]
```diff
@@ -210,6 +210,16 @@ function save_set(setname)
     notice('Set '..setname..' saved.')
 end
 
+function delete_set(setname)
+	if settings.spellsets[setname] == nil then
```
Comment (Chiaia) [inline addons/azureSets/azuresets.lua]: Need to replace the tab with spaces.

File: addons/azureSets/azuresets.lua [hunk 2]
```diff
@@ -248,7 +258,10 @@ windower.register_event('addon command', function(...)
             if args[1] ~= nil then
                 save_set(args[1])
             end
-
+		elseif comm == 'delete' then
```
Comment (Chiaia) [inline addons/azureSets/azuresets.lua]: Need to replace the tabs with spaces.

File: addons/azureSets/azuresets.lua (hunk 1 above)
Comment (DYDR) [reply addons/azureSets/azuresets.lua]: Updated

File: addons/azureSets/azuresets.lua (hunk 2 above)
Comment (DYDR) [reply addons/azureSets/azuresets.lua]: Updated

**Comment by Chiaia** [comment]
Also needs a version bump then it's good to go.

**Pull request merged**